### PR TITLE
Fix ProtectedModel

### PR DESCRIPTION
**Pull request by dyve**
ProtectedModel had a serious bug, a call to a non-existing `super().update()`.

### CODE REVIEW

File: tests/test_db.py [hunk 1]
```diff
@@ -7,6 +7,25 @@
 
 
 class ProtectedModelMixinTestCase(TestCase):
+    def test_update_protected(self):
+        protected = ProtectedModel(name="protected", is_update_protected=True)
+        protected.save()
+        with self.assertRaises(PermissionDenied):
+            protected.save()
```
Comment (jieter) [inline tests/test_db.py:14]: Why not use 
```suggestion
        with self.assertRaisesMessage(PermissionDenied, "This object has update protection."):
            protected.save()
```

and leave out `test_get_protected_against_update_message`?

Comment (dyve) [reply tests/test_db.py:14]: Atomic testing, `get_protected_against_update_message` is a public method, should be tested separately.

**Comment by dyve** [comment]
My main concern here was the "brutal blocking" of the save() method.

File: tests/test_db.py (hunk 1 above)
Comment (jieter) [reply tests/test_db.py:14]: ok, fair enough!

**Review by jieter — APPROVED** [review]

**Pull request merged**